app/io/pathfinder/controllers/PreFlight.java

```diff
@@ -0,0 +1,18 @@
+package io.pathfinder.controllers;
+
+import play.mvc.*;
```

Comment:
Please don't use wildcard imports, it makes it harder for other people to figure out what the code is doing.